sql/database.sql

```diff
@@ -2662,9 +2662,9 @@ DROP TABLE IF EXISTS `lang_custom`;
 CREATE TABLE `lang_custom` (
   `lang_description` varchar(100) NOT NULL default '',
   `lang_code` char(2) NOT NULL default '',
-  `constant_name` mediumtext NOT NULL default '',
-  `definition` mediumtext NOT NULL default ''
-) ENGINE=MyISAM ;
+  `constant_name` mediumtext NOT NULL,
+  `definition` mediumtext
```

Comment:
So, we basically can not set TEXT fields to default?
So, they will always then be NULL if not set?